Dockerfile.fpm

```diff
@@ -11,10 +15,10 @@ RUN composer install
 # yes this is how you copy directories
 COPY bin/ /srv/annotations/bin/
 COPY config/ /srv/annotations/config/
-COPY phpunit.xml.dist /srv/annotations/
 COPY src/ /srv/annotations/src/
 COPY tests/ /srv/annotations/tests/
```

Comment:
Not needed if `phpunit.xml.dist` is being removed?

Reply:
I think the `Dockerfile.ci` file could do much of the tests set up in place of this

Reply:
👍  Uses `Dockerfile.cli` so definitely not needed here, but `Dockerfile.ci` should add `tests/` itself anyway (so CLI and FPM match).